utils/cli-utils/Cargo.toml

```
@@ -10,14 +10,17 @@ description = "CLI utils"
 documentation = "https://docs.rs/cli-utils"
 
 [dependencies]
-frame-system = { version = "2.0.0-alpha.4", default-features = false, path = "../../frame/system" }
-pallet-balances = { version = "2.0.0-alpha.4", default-features = false, path = "../../frame/balances" }
-sp-runtime = { version = "2.0.0-alpha.4", default-features = false, path = "../../primitives/runtime" }
+frame-system = { version = "2.0.0-alpha.7", default-features = false, path = "../../frame/system" }
+pallet-balances = { version = "2.0.0-alpha.7", default-features = false, path = "../../frame/balances" }
+sp-runtime = { version = "2.0.0-alpha.7", default-features = false, path = "../../primitives/runtime" }
+sp-core = { version = "2.0.0-alpha.7", default-features = false, path = "../../primitives/core" }
+codec = { version = "1.3.0", default-features = false, package = "parity-scale-codec" }
 
 [features]
 default = ["std"]
 std = [
     "frame-system/std",
     "pallet-balances/std",
     "sp-runtime/std",
+	"sp-core/std",
```

don't ask me why, just keeping it consistent.
```suggestion
    "sp-core/std",
```